cllc-public-app/ClientApp/src/app/app.component.html

```diff
@@ -1,7 +1,9 @@
-<div *ngIf="featureFlagService.initialized">
-  <section class="container not-supported-browser" *ngIf="isIE()">
-    <p>Internet Explorer is not supported</p>
-    <p>Please use Google Chrome, Firefox or Microsoft Edge</p>
+<div *ngIf="featureFlagService.initialized" *ngIf="isIE()">

```

Comment:
Two ngIf attributes on the same tag may cause a problem

Reply:
I have made the change, thanks for your review